Gems/ROS2/Code/Source/Lidar/ROS2LidarSensorComponent.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "Lidar/LidarRaycaster.h"
+#include "Lidar/LidarTemplate.h"
+#include "Lidar/LidarTemplateUtils.h"
+#include "ROS2/Sensor/ROS2SensorComponent.h"
```

Comment:
The Lidar includes should remove the "Lidar/" portion of the path while ROS2 should use angle brackets.
```suggestion
#include "LidarRaycaster.h"
#include "LidarTemplate.h"
#include "LidarTemplateUtils.h"
#include <ROS2/Sensor/ROS2SensorComponent.h>
```